documentation/build_docs.py

```diff
@@ -11,15 +11,22 @@
 from subprocess import Popen
 
 docs_dir = os.path.dirname(os.path.abspath(__file__))
+version_file = os.path.join(docs_dir, os.pardir, 'agentos', 'version.py')
```

Comment:
Good idea!

Is `os.pardir` relative to the file it’s used in or the path where the script is run from? If it’s the latter, then this wouldn’t run from outside the documentation dir any more. Did you test that by any chance? I had a similar issue before which is why I had added the `docs_dir` variable.

I created issue #47 — create a test that attempts to build the docs.

Reply:
Duh, nevermind this is exactly what you’re doing!